scripts/dataset_processing/tts/compute_features.py

```diff
@@ -0,0 +1,184 @@
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This script computes features for TTS models prior to training, such as pitch and energy.
```

Comment:
what is the relationship between this script and https://github.com/NVIDIA/NeMo/blob/vits_doc/scripts/dataset_processing/tts/extract_sup_data.py? If we are planning to deprecate `extract_sup_data.py`, shall we add a note here or there?

Reply:
I imagine we will deprecate the current TTSDataset (and extract_sup_data.py which relies on it), but do not want to assume that until the new data loader is finished and tested by more people.